plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/ColumnWithAliasFormatter.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import io.trino.spi.connector.ConnectorSession;
+
+import static com.google.common.base.Splitter.fixedLength;
+import static com.google.common.base.Strings.padStart;
+
+public class ColumnWithAliasFormatter
+{
+    public static final int DEFAULT_COLUMN_ALIAS_LENGTH = 30;
+    public static final int ORIGINAL_COLUMN_NAME_LENGTH = 24;
```

Comment:
> the code lives in trino-base-jdbc but the values are specific to Oracle.

It's ok to ignore for now since as follow-up we'll make this general for all connectors.

Reply:
> Also consider adding a explanatory comment about why ORIGINAL_COLUMN_NAME_LENGTH = DEFAULT_COLUMN_ALIAS_LENGTH - 6. Where does the magic value 6 come from?

This still applies.